templates/landscape/install.html

```diff
@@ -448,9 +250,10 @@ <h3 class="p-heading--2">Configure postfix for email</h3>
               <ol class="p-list--divided">
                 <li class="p-list__item">
                   <p class="u-no-padding--top">Set SMTP information in variables</p>
+                  <p>Use an API Key from: https://app.sendgrid.com/settings/api_keys</p>
                   <div class="p-code-snippet">
                     <pre class="p-code-snippet__block is-wrapped"><code>SMTP_HOST='smtp.sendgrid.net'
-SMTP_PORT='465'
+SMTP_PORT='587'
```

Comment:
below, both of the following list items "Download an SSL CA file" and "Set appropriate permissions" need to be removed